legend-engine-xts-persistence/legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-relational-snowflake/pom.xml

```diff
@@ -88,7 +88,7 @@
             <groupId>net.snowflake</groupId>
             <artifactId>snowflake-jdbc</artifactId>
             <version>3.13.19</version>
-            <scope>runtime</scope>
```

Comment:
We need to keep it run time so the application using the lib can provide it

Reply:
we need SnowflakeResultSet for compilation, moved to provided